crates/autopilot/src/domain/settlement/solution/trade.rs

```diff
@@ -437,6 +438,26 @@ impl Trade {
             order::Side::Sell => self.buy.token,
         }
     }
+
+    /// Protocol defines rules whether a trade is eligible to contribute to the
+    /// surplus of a settlement.
+    ///
+    /// This property is always evaluated in the context of an auction
+    /// accociated to a settlement.
+    fn is_surplus_capturing(&self, auction: &settlement::Auction) -> bool {
```

Comment:
nit: I would expose this on the auction as we might not only want to use it from observed `settlement::trade` but also from proposed trades as part of the `/solve` response and it feels a bit more natural to ask the auction is a speicific order id should capture surplus.